modular_zubbers/code/modules/jobs/stowaway.dm

```diff
@@ -0,0 +1,42 @@
+/datum/job/assistant/stowaway
+	title = "Stowaway"
+	description = "Be a stowaway crewmember."
```

Comment:
Should change this description to make it clear that you're not crew and don't count as one.

Reply:
done